src/pages/workspace/WorkspaceNewRoomPage.js

```diff
@@ -143,59 +110,59 @@ class WorkspaceNewRoomPage extends React.Component {
             return null;
         }
 
+        // Workspaces are policies with type === 'free'
+        const workspaceOptions = _.map(
+            _.filter(this.props.policies, policy => policy && policy.type === CONST.POLICY.TYPE.FREE),
+            policy => ({label: policy.name, key: policy.id, value: policy.id}),
+        );
+
         const visibilityOptions = _.map(_.values(CONST.REPORT.VISIBILITY), visibilityOption => ({
             label: this.props.translate(`newRoomPage.visibilityOptions.${visibilityOption}`),
             value: visibilityOption,
             description: this.props.translate(`newRoomPage.${visibilityOption}Description`),
         }));
 
         return (
-            <ScreenWrapper onTransitionEnd={this.focusRoomNameInput}>
+            <ScreenWrapper>
```

Comment:
Is there any reason to remove auto focus here?

Reply:
I feel like we had a convo about this, but I don't recall why it was done. @fedirjh do you?